src/main/charts/bitbucket/templates/service.yaml

```diff
@@ -22,3 +26,27 @@ spec:
       name: hazelcast
   selector:
     {{- include "bitbucket.selectorLabels" . | nindent 4 }}
+
+{{/* This describes a K8s service for communicating with the ssh port on Bitbucket */}}
```

Comment:
To be consistent with other charts (e.g. synchrony in confluence) I believe this service should be defined in a separate file - `service-ssh.yaml` so it is clearer for anybody who browses the repo.